### ISSUE TITLE
[FE] Retrieve project state number values (shouldn't be defined manually)

### ISSUE THREAD

**Issue by BenasVolkovas** [buidlone/frontend#89]

**Comment by matasvosylius** [comment]
#81 duplicate? @BenasVolkovas

**Comment by BenasVolkovas** [comment]
No. In the previous issue,  frontend needed to know the exact values and hardcode them. But with the functions above, frontend can save those values for later usage and without knowing the actual numbers, use them.

For example this code:
`
if (**[4, 32]**.includes(projectState) && totalInvested !== hardCap) {
    return true;
}
`
Should be changed to:
`
const activeFundraiserState = await contract.ACTIVE_FUNDRAISER_BYTE_VALUE();
const notLastActiveMilestoneState = await contract.NOT_LAST_ACTIVE_MILESTONE_BYTE_VALUE();

if (**[activeFundraiserState, notLastActiveMilestoneState]**.includes(projectState) && totalInvested !== hardCap) {
    return true;
}
`
